src/transformers/trainer.py

```diff
@@ -1717,7 +1731,6 @@ def _inner_training_loop(
 
             step = -1
             for step, inputs in enumerate(epoch_iterator):
-
```

Comment:
Please let's not add unrelated changes like all those empty lines removed.

Reply:
done